modules/ROOT/pages/index.adoc

```diff
@@ -1,88 +1,132 @@
-Anypoint Platform makes it easy to discover, create, and manage APIs in a modular, organized layer. Instead of retrieving random (and possibly unstable) code snippets, you can discover and use APIs that are created using the industry’s best practices.
+--
+[discrete]
+=== Getting Started
 
-== Exploring Anypoint Exchange
+Build the digital transformation your business needs.
 
-The APIs you build in Anypoint Platform to integrate applications and services are reusable and include enterprise-level security.
-You can discover these APIs, as well as connectors, samples, and templates, in
-https://anypoint.mulesoft.com/exchange/[Exchange^].
+* xref:api-led-overview.adoc[Tutorial: Build an API from Start to Finish]
+--
 
-Exchange also provides RAML fragments, custom packages, videos, links to documentation, and other assets.
+image::discover.png[]
 
-Discover Anypoint Platform APIs on https://anypoint.mulesoft.com/exchange/portals/anypoint-platform/[the MuleSoft developer portal^].
+--
+[discrete]
+=== Discover
 
-== Designing and Building API-Based Integrations
+Browse Exchange to find existing API specifications, templates, examples, and other assets that you can reuse for your projects.
 
-Anypoint Studio is an Eclipse-based IDE that enables you to create integrations. Design Center, a web-based tool, enables you to create API specifications, which are the foundation for these integrations.
+* https://www.anypoint.mulesoft.com/exchange/[Discover reusable assets^]
+--
 
-[%header%autowidth.spread]
-|===
-| Creating Integrations | Creating API Specifications
-| xref:studio::index.adoc[Building Integrations with Anypoint Studio] | xref:studio::import-api-specification-exchange.adoc[Importing APIs from Exchange to Studio]
-| xref:design-center::about-designing-a-mule-application.adoc[Building Integrations in Design Center]  | xref:design-center::design-create-publish-api-specs.adoc[Creating API specifications in Design Center]
-|===
+image::design.png[]
 
-Anypoint Studio 7.0 and later includes xref:mule-runtime::index.adoc[Mule runtime engine (Mule) 4], the runtime environment in which you deploy Mule apps and APIs.
-Earlier versions of Studio include xref:3.9@mule-runtime::index.adoc[Mule 3].
+--
+[discrete]
+=== Design
 
-[%header%autowidth.spread]
-|===
-| Mule 3    | Mule 4
-| xref:3.9@mule-runtime::anypoint-connectors.adoc[Connectors for Mule 3] | xref:connectors::index.adoc[Connectors for Mule 4]
-| xref:3.9@mule-runtime::dataweave.adoc[DataWeave 1 for Mule 3]  | xref:mule-runtime::dataweave.adoc[DataWeave 2 for Mule 4]
-| xref:3.x@apikit::index.adoc[APIkit 3] | xref:4.x@apikit::overview-4.adoc[APIkit 4]
-| xref:3.9@connector-devkit::index.adoc[Connector DevKit] | xref:1.1@mule-sdk::index.adoc[Mule SDK]
-|===
+Create and publish new API specifications from scratch.
+
+* xref:design-center::design-create-publish-api-specs.adoc[Create a reusable API specification]
```

Comment:
Do we want to tie these tasks to products? 
Create a reusable API specification with API Designer.

Reply:
My only concern with this approach is that some of these tasks are already long enough to add the product's name. Also, does it add any value for the user to mention the product's name here?

Reply:
I guess it's not necessary.